src/test/java/emissary/util/shell/ExecutrixTest.java

```diff
@@ -96,180 +96,180 @@ public void testExecutrixUniqueBase() {
 
         data = Executrix.readDataFromChannel(channel, 0, 3);
-        assertNotNull("Available data read from channel", data);
-        assertTrue("All available data read from channel", data.length == 3);
+        assertNotNull(data, "Available data read from channel");
+        assertEquals(3, data.length, "All available data read from channel");
 
         data = Executrix.readDataFromChannel(channel, 0, 1);
-        assertNotNull("Available data read from channel", data);
-        assertTrue("All limit data read from channel", data.length == 1);
+        assertNotNull(data, "Available data read from channel");
+        assertEquals(1, data.length, "All limit data read from channel");
 
         data = Executrix.readDataFromChannel(channel, 0, 100);
-        assertNotNull("Available data read from channel", data);
-        assertTrue("All limit data read from channel", data.length == 3);
+        assertNotNull(data, "Available data read from channel");
+        assertEquals(3, data.length, "All limit data read from channel");
 
         data = Executrix.readDataFromChannel(channel, 50, 100);
-        assertNull("Out of bounds data on channel returned non-null", data);
+        assertNull(data, "Out of bounds data on channel returned non-null");
 
         channel.close();
         raf.close();
 
         final File f = new File(TMPDIR + "/foo.dat");
-        f.delete();
+        Files.deleteIfExists(f.toPath());
     }
 
     @Test
-    public void testReadWriteTempDir() {
+    void testReadWriteTempDir() throws IOException {
         String[] names = this.e.writeDataToNewTempDir("aaa".getBytes());
-        assertNotNull("names on temp dir write", names);
+        assertNotNull(names, "names on temp dir write");
         readAndNuke(names[Executrix.INPATH]);
         Executrix.cleanupDirectory(names[Executrix.DIR]);
 
         names = this.e.writeDataToNewTempDir("aaa".getBytes(), 0, 1);
-        assertNotNull("names on temp dir write", names);
+        assertNotNull(names, "names on temp dir write");
         readAndNuke(names[Executrix.INPATH]);
         Executrix.cleanupDirectory(names[Executrix.DIR]);
     }
 
     @Test
-    public void testCopyFile() throws Exception {
+    void testCopyFile() throws Exception {
         final String TMPDIR = this.e.getTmpDir();
-        assertTrue("File written", Executrix.writeDataToFile("aaa".getBytes(), 0, 3, TMPDIR + "/foo.dat", false));
+        assertTrue(Executrix.writeDataToFile("aaa".getBytes(), 0, 3, TMPDIR + "/foo.dat", false), "File written");
         Executrix.copyFile(TMPDIR + "/foo.dat", TMPDIR + "/bar.dat");
         final byte[] data = Executrix.readFile(TMPDIR + "/bar.dat");
-        assertNotNull("Data read from copy", data);
-        assertTrue("All data read from copy", data.length == 3);
+        assertNotNull(data, "Data read from copy");
+        assertEquals(3, data.length, "All data read from copy");
 
         File f = new File(TMPDIR + "/foo.dat");
```

Comment:
(Outside the PR scope, but these cleanup statements should be in a `finally` block because they won't be executed if an earlier assertion fails.)